### PR TITLE
Make compatible for k8s 1 17

### PR DESCRIPTION
**Pull request by nandodelacruz**

### CODE REVIEW

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-90**

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-90**

**Review by jason-shiao — DISMISSED** [review]

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-90**

**Review by hoaphanFox — APPROVED** [review]

File: charts/preview/values.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ netflix-conductor:
     loadSamples: true
 #    podAnnotations:
 #     iam.amazonaws.com/role: conductor-role
-    replicaCount: 2
+    replicaCount: 1
```
Comment (hoaphanFox) [inline charts/preview/values.yaml:32]: noice

**Review by hoaphanFox — APPROVED** [review]

**Review by javierazabache — APPROVED** [review]

**Pull request merged**